### PR TITLE
Feed fix

### PR DESCRIPTION
**Pull request by gauntface**
@petele @PaulKinlan 

This fixes up the title and descriptions missing from the feeds.

Do we want feeds for tools and showcases section?

### CODE REVIEW

**Comment by PaulKinlan** [comment]
Showcases: Yes.
Tools: Unsure.

Related issue for this fix #2190

File: src/jekyll/_plugins/custom_tags/ytvideo.rb [hunk 1]
```diff
@@ -54,9 +54,14 @@ def render(context)
...
       end
 
-      out = '<div class="embed">'
-      out += '<iframe src="' + iframeSrcUrl + '" frameborder="0" allowfullscreen=""></iframe>'
-      out += "</div>"
+      #out = '<div class="embed">'
+      #out += '<iframe src="' + iframeSrcUrl + '" frameborder="0" allowfullscreen=""></iframe>'
+      #out += "</div>"
+      out =  '<div class="video-wrapper">'
+      out += '<iframe src="' + iframeSrcUrl + '" class="devsite-embedded-youtube-video" data-video-id="dQw4w9WgXcQ" '
```
Comment (petele) [inline src/jekyll/_plugins/custom_tags/ytvideo.rb]: did you mean to hard code the `data-video-id` here?

File: src/jekyll/_plugins/custom_tags/ytvideo.rb [hunk 2]
```diff
@@ -54,9 +54,14 @@ def render(context)
         iframeSrcUrl += '&amp;' + argument
       end
 
-      out = '<div class="embed">'
-      out += '<iframe src="' + iframeSrcUrl + '" frameborder="0" allowfullscreen=""></iframe>'
-      out += "</div>"
+      #out = '<div class="embed">'
```
Comment (petele) [inline src/jekyll/_plugins/custom_tags/ytvideo.rb]: Unless we really want to keep these commented out lines, let's yank them completely.

**Comment by PaulKinlan** [comment]
All looks good with this.

**Pull request merged**